2024-01-08 15:01:48,593 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-08 15:01:48,594 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Configure stats pid to 35222
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from /home/users/maali/.config/wandb/settings
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from /mnt/irisgpfs/users/maali/wandb/settings
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_init.py:_log_setup():524] Logging user logs to /home/users/maali/TRAFFICLIGHTDETECTION/wandb/run-20240108_150148-73oggbwr/logs/debug.log
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_init.py:_log_setup():525] Logging internal logs to /home/users/maali/TRAFFICLIGHTDETECTION/wandb/run-20240108_150148-73oggbwr/logs/debug-internal.log
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_init.py:init():564] calling init triggers
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-01-08 15:01:48,599 INFO    MainThread:35222 [wandb_init.py:init():589] re-initializing run, found existing run on stack: 4ywooyoz
2024-01-08 15:01:48,601 INFO    MainThread:35222 [wandb_run.py:_finish():1962] finishing run indi-team/uncategorized/4ywooyoz
2024-01-08 15:01:48,602 INFO    MainThread:35222 [jupyter.py:save_history():445] not saving jupyter history
2024-01-08 15:01:48,602 INFO    MainThread:35222 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-08 15:01:48,602 INFO    MainThread:35222 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-01-08 15:01:48,602 INFO    MainThread:35222 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-08 15:01:48,602 INFO    MainThread:35222 [wandb_run.py:_restore():2185] restore
2024-01-08 15:01:48,602 INFO    MainThread:35222 [wandb_run.py:_restore():2191] restore done
2024-01-08 15:01:52,003 INFO    MainThread:35222 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-08 15:01:52,004 INFO    MainThread:35222 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-08 15:01:52,007 INFO    MainThread:35222 [wandb_run.py:_footer_sync_info():3796] logging synced files
2024-01-08 15:01:52,052 INFO    MainThread:35222 [wandb_init.py:init():614] starting backend
2024-01-08 15:01:52,052 INFO    MainThread:35222 [wandb_init.py:init():618] setting up manager
2024-01-08 15:01:52,054 INFO    MainThread:35222 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-08 15:01:52,055 INFO    MainThread:35222 [wandb_init.py:init():624] backend started and connected
2024-01-08 15:01:52,062 INFO    MainThread:35222 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-08 15:01:52,062 INFO    MainThread:35222 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-08 15:01:52,062 INFO    MainThread:35222 [wandb_init.py:init():716] updated telemetry
2024-01-08 15:01:52,063 INFO    MainThread:35222 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-01-08 15:01:53,357 INFO    MainThread:35222 [wandb_run.py:_on_init():2254] communicating current version
2024-01-08 15:01:53,401 INFO    MainThread:35222 [wandb_run.py:_on_init():2263] got version response 
2024-01-08 15:01:53,401 INFO    MainThread:35222 [wandb_init.py:init():800] starting run threads in backend
2024-01-08 15:01:53,458 INFO    MainThread:35222 [wandb_run.py:_console_start():2233] atexit reg
2024-01-08 15:01:53,458 INFO    MainThread:35222 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-08 15:01:53,459 INFO    MainThread:35222 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-08 15:01:53,459 INFO    MainThread:35222 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-08 15:01:53,461 INFO    MainThread:35222 [wandb_init.py:init():841] run started, returning control to user process
2024-01-08 15:12:22,522 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Configure stats pid to 35222
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from /home/users/maali/.config/wandb/settings
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from /mnt/irisgpfs/users/maali/wandb/settings
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_init.py:_log_setup():524] Logging user logs to /home/users/maali/TRAFFICLIGHTDETECTION/wandb/run-20240108_151222-lgm6gz4u/logs/debug.log
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_init.py:_log_setup():525] Logging internal logs to /home/users/maali/TRAFFICLIGHTDETECTION/wandb/run-20240108_151222-lgm6gz4u/logs/debug-internal.log
2024-01-08 15:12:22,524 INFO    MainThread:35222 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ffeb1bb9370>
2024-01-08 15:12:22,526 INFO    MainThread:35222 [wandb_init.py:init():564] calling init triggers
2024-01-08 15:12:22,526 INFO    MainThread:35222 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-01-08 15:12:22,526 INFO    MainThread:35222 [wandb_init.py:init():589] re-initializing run, found existing run on stack: 73oggbwr
2024-01-08 15:12:22,528 INFO    MainThread:35222 [wandb_run.py:_finish():1962] finishing run indi-team/uncategorized/73oggbwr
2024-01-08 15:12:22,528 INFO    MainThread:35222 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-08 15:12:22,529 INFO    MainThread:35222 [wandb_run.py:_restore():2185] restore
2024-01-08 15:12:22,529 INFO    MainThread:35222 [wandb_run.py:_restore():2191] restore done
2024-01-08 15:12:28,655 INFO    MainThread:35222 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-08 15:12:28,655 INFO    MainThread:35222 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-08 15:12:28,660 INFO    MainThread:35222 [wandb_run.py:_footer_sync_info():3796] logging synced files
